go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/k8snetworkplumbingwg/sriov-network-operator
 
-go 1.20
+go 1.21
```

Comment:
do you need to update all relevant github action workflows as well ?

Reply:
yes we need

Reply:
Agreed. I've bumped the version in GitHub actions and also in all Dockerfiles